pinecone/core/client/__init__.py

```diff
@@ -1,10 +1,6 @@
-#
-# Copyright (c) 2020-2021 Pinecone Systems Inc. All right reserved.
-#
 
-# flake8: noqa
 
-"""
+# flake8: noqa"""
```

Comment:
I think the beginning of the comment block needs to drop to the following line. This PR is failing testing in CI due to IndentationErrors in this file, looks like:

> ==================================== ERRORS ====================================
  __________________ ERROR collecting tests/unit/test_config.py __________________
  .tox/py38/lib/python3.8/site-packages/_pytest/python.py:578: in _importtestmodule
      mod = import_path(self.fspath, mode=importmode)
  .tox/py38/lib/python3.8/site-packages/_pytest/pathlib.py:524: in import_path
      importlib.import_module(module_name)
  /opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/importlib/__init__.py:127: in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
  <frozen importlib._bootstrap>:1014: in _gcd_import
      ???
  <frozen importlib._bootstrap>:991: in _find_and_load
      ???
  <frozen importlib._bootstrap>:975: in _find_and_load_unlocked
      ???
  <frozen importlib._bootstrap>:671: in _load_unlocked
      ???
  .tox/py[38](https://github.com/pinecone-io/pinecone-python-client/actions/runs/6312132763/job/17137537015#step:5:39)/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:170: in exec_module
      exec(co, module.__dict__)
  tests/unit/test_config.py:1: in <module>
      import pinecone
  pinecone/__init__.py:2: in <module>
      from .config import *
  pinecone/config.py:13: in <module>
      from pinecone.core.client.exceptions import ApiKeyError
  E     File "/home/runner/work/pinecone-python-client/pinecone-python-client/pinecone/core/client/__init__.py", line 4
  E       Pinecone API
  E       ^
  E   IndentationError: unexpected indent